prboom2/src/r_bsp.c

```diff
@@ -667,7 +679,7 @@ static void R_Subsector(int num)
   // e6y
   // New algo can handle fake flats and ceilings
   // much more correctly and fastly the the original
-    if (V_IsOpenGLMode())
+    if (V_IsOpenGLMode() && sub->sector->gl_validcount != validcount)
```

Comment:
Why was this happening every time before and now should only happen once?

Reply:
It was never necessary to do it every time before, and operating on potentially stale values of `floorplane/ceilingplane` made me nervous.